lib/systems/examples.nix

```
@@ -329,6 +329,6 @@ rec {
 
   # Ghcjs
   ghcjs = {
-    config = "js-unknown-ghcjs";
+    config = "javascript-unknown-ghcjs";
```

```suggestion
    config = "asmjs-ghc-none";
```

`gnu-config` doesn't recognize `js-unknown-ghcjs`, so if we're going to change the tuple maybe now is a good time to switch to something it *does* recognize?

`gnu-config` parses this as `cpu=asmjs`, `vendor=ghc`, `kernel=null`, `environment=ghc`

(See also my next comment in the documentation)